client/blocks/image-editor/image-editor-crop.jsx

```diff
@@ -102,7 +103,10 @@ class ImageEditorCrop extends Component {
 			} );
 		}
 
-		if ( aspectRatio !== newProps.aspectRatio ) {
+		if (
+			( aspectRatio !== newProps.aspectRatio ) ||
+			! this.state.wasCropUpdated
```

Comment:
What happens when if we don't check for `wasCropUpdated`? Is there any way to avoid setting state for this case by comparing current to last props?